### PR TITLE
Port robotiq_3_finger_gripper_driver to ROS 2

### PR DESCRIPTION
**Pull request by hidmic**
Precisely what the title says. Fifth PR of a series of 9 PRs -- one per package in this repository.

This patch adds ROS 2 support alongside ROS 1 support, just like ToyotaResearchInstitute/common_robotics_utilities does.
Depends on https://github.com/calderpg/common_robotics_utilities/pull/18.

### CODE REVIEW

**Comment by hidmic** [comment]
FYI @calderpg-tri

**Review by calderpg-tri — APPROVED** [review]

**Comment by calderpg-tri** [comment]
I intend to wait to merge this until all of the non-UR drivers are ready and all the PRs can be merged at once, if that's OK with you.

**Comment by hidmic** [comment]
Updated to improve Bazel users' UX.

File: robotiq_3_finger_gripper_driver/src/robotiq_3_finger_gripper_driver_node_main.ros2.cpp [hunk 1]
```diff
@@ -0,0 +1,16 @@
+#include <memory>
+
+#include <rclcpp/rclcpp.hpp>
+
+#include "robotiq_3_finger_gripper_driver_node.ros2.hpp"
+
+int main(int argc, char * argv[])
```
Comment (calderpg-tri) [inline robotiq_3_finger_gripper_driver/src/robotiq_3_finger_gripper_driver_node_main.ros2.cpp:7]: This should be `char** argv` to be consistent with existing code

**Review by calderpg-tri — CHANGES_REQUESTED** [review]

File: robotiq_3_finger_gripper_driver/src/robotiq_3_finger_gripper_driver_node_main.ros2.cpp (hunk 1 above)
Comment (hidmic) [reply robotiq_3_finger_gripper_driver/src/robotiq_3_finger_gripper_driver_node_main.ros2.cpp]: Done in https://github.com/ToyotaResearchInstitute/tri_hardware_drivers/pull/14/commits/349ad4728d2f4bd362c7a8971c81102748c14663.

**Review by calderpg-tri — APPROVED** [review]

**Pull request merged**